loadfile/loadfile_test.go

```diff
@@ -15,12 +15,12 @@ import (
 // engine should only attempt to read files of a type that the os can
 // read (i.e. not throw an error on a call to os.ReadFile()), and
 // disregard (not throw an error) files with a type it cannot read.
-func TestLoadContext(t *testing.T) {
+func Test_LoadContext(t *testing.T) {
 	testdir := "/tmp/loadfile-test"
-	// cleanup directory even if it's there
-	_ = os.RemoveAll(testdir)
-
 	assert.NoError(t, os.MkdirAll(testdir, os.ModePerm))
+	t.Cleanup(func() {
+		assert.NoError(t, os.RemoveAll(testdir))
+	})
```

Comment:
So you moved this to the cleanup?
Theoretically, would this cause a problem if the last test failed to clean up?

Reply:
yeah it could; I added directory cleanup into `TestMain()`